module/chunks/chunkVerifier_test.go

```diff
@@ -136,14 +155,34 @@ func (s *ChunkVerifierTestSuite) TestFailedTx() {
 // TestEventsMismatch tests verification behavior in case
 // of emitted events not matching chunks
 func (s *ChunkVerifierTestSuite) TestEventsMismatch() {
-	vch := GetBaselineVerifiableChunk(s.T(), "eventsMismatch")
+	vch := GetBaselineVerifiableChunk(s.T(), "eventsMismatch", false)
 	assert.NotNil(s.T(), vch)
 	_, chFault, err := s.verifier.Verify(vch)
 	assert.Nil(s.T(), err)
 	assert.NotNil(s.T(), chFault)
 	assert.IsType(s.T(), &chunksmodels.CFInvalidEventsCollection{}, chFault)
 }
 
+// TestServiceEventsMismatch tests verification behavior in case
+// of emitted service events not matching chunks'
+func (s *ChunkVerifierTestSuite) TestServiceEventsMismatch() {
+	vch := GetBaselineVerifiableChunk(s.T(), "serviceEventsMismatch", true)
+	assert.NotNil(s.T(), vch)
+	_, chFault, err := s.systemBadVerifier.SystemChunkVerify(vch)
+	assert.Nil(s.T(), err)
+	assert.NotNil(s.T(), chFault)
+	assert.IsType(s.T(), &chunksmodels.CFInvalidSystemEventsEmitted{}, chFault)
+}
+
+// TestServiceEventsAreChecked ensures that service events are in fact checked
+func (s *ChunkVerifierTestSuite) TestServiceEventsAreChecked() {
+	vch := GetBaselineVerifiableChunk(s.T(), "serviceEventsMismatch", true)
```

Comment:
```suggestion
	vch := GetBaselineVerifiableChunk(s.T(), "serviceEventsAreChecked", true)
```

I'm guessing we want the script to match the test here

Reply:
It doesn't matter really, but better to keep it right